src/main/java/teammates/ui/controller/AdminEmailLogPageAction.java

```diff
@@ -113,7 +113,6 @@ protected ActionResult execute() {
 
         String status = "&nbsp;&nbsp;Total Logs gone through in last search: "
                       + totalLogsSearched + "<br>"
-        //link for Next button, will fetch older logs
```

Comment:
It's quite obvious as the statement already shows what is going on. The comment is useless.